src/main/java/graphql/execution/instrumentation/tracing/TracingSupport.java

```diff
@@ -26,6 +26,8 @@
     private final Instant startRequestTime;
     private final long startRequestNanos;
     private final ConcurrentLinkedQueue<Map<String, Object>> fieldData;
+    private final Map<String, Object> parseMap = new LinkedHashMap<>();
+    private final Map<String, Object> validationMap = new LinkedHashMap<>();
 
```

Comment:
Can you please make this ConcurrentHashMap.

I know that parse and validation are on the one thread and concurrent but for consistency can you please make it so